STMicroelectronics/STM32L4_L4+/app/startup/gnu/STM32L4S5xx_FLASH.ld

```diff
@@ -64,9 +64,15 @@ __RAM_segment_used_end__ = 0;
 MEMORY
 {
 RAM (xrw)      : ORIGIN = 0x20000000, LENGTH = 640K
-FLASH (rx)      : ORIGIN = 0x8000000, LENGTH = 2048K
+FLASH (rx)      : ORIGIN = 0x8000000, LENGTH = 2044K
+REGISTRYINFO (rx)      : ORIGIN = 0x80FD800, LENGTH = 2K
+REGISTRY (rx)      : ORIGIN = 0x80FF000, LENGTH = 4K
```

Comment:
out of my wheelhouse here, but wouldn't 6K of room in `FLASH` have to be made to accomodate `REGISTRYINFO` and `REGISTRY`? (i.e.) 
```
FLASH (rx)      : ORIGIN = 0x8000000, LENGTH = 2042K
```

Reply:
good catch, forgot to update this linker file. thanks!